src/assignmentB/AssignmentB02.java

```diff
@@ -1,5 +1,96 @@
 package assignmentB;
 
-public class AssignmentB02
+import javax.swing.*;
+import javax.swing.table.DefaultTableModel;
+import java.awt.*;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.Locale;
+
+public class AssignmentB02 extends JFrame
 {
+    Calendar cal = new GregorianCalendar();
+    DefaultTableModel model;
+    JLabel label;
+
+    //TODO: add text box to add/view appointments
+
+    AssignmentB02()
+    {
+        this.setTitle("Appointment Calendar");
+        this.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        this.setSize(640, 480);
+        this.setVisible(true);
+
+        //TODO: fix formatting... need fixed table that doesn't resize with window
+        // BoarderLayout() ??
+        this.setLayout(new FlowLayout());
+
+        label = new JLabel();
+        label.setHorizontalAlignment(SwingConstants.CENTER);
+
+        JButton prevMonth = new JButton("Prev");
+        JButton nextMonth = new JButton("Next");
+
+        prevMonth.addActionListener(ae -> {
+            cal.add(Calendar.MONTH, -1);
+            updateMonth();
+        });
+
+        nextMonth.addActionListener(ae -> {
+            cal.add(Calendar.MONTH, +1);
+            updateMonth();
+        });
+
+        JPanel panel = new JPanel();
+        panel.setLayout(new FlowLayout());
+        panel.add(prevMonth, FlowLayout.LEFT);
+        panel.add(label, BorderLayout.CENTER);
+        panel.add(nextMonth, FlowLayout.RIGHT);
+
+        String[] days = {"Sun","Mon","Tue","Wed","Thu","Fri","Sat"};
+
+        model = new DefaultTableModel(null, days)
+        {
+            @Override
+            public boolean isCellEditable(int row, int column) {
+                return false;
+            }
+        };
+
+        JTable table = new JTable(model);
+        JScrollPane pane = new JScrollPane(table);
+        this.add(panel,BorderLayout.NORTH);
+        this.add(pane,BorderLayout.CENTER);
+
+        this.updateMonth();
+    }
+
+    void updateMonth()
```

Comment:
A more descriptive name for this method could be `updateCalendarState`.

Reply:
name changed to updateCalendarState